CHANGELOG.md

```diff
@@ -10,9 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add events `meetingReconnected`, `signalingDropped` and `receivingAudioDropped` to `eventDidReceive` by publishing them as stand alone events. Currently, these events were only included in the meeting history attribute when a meeting event is published. 
 
+### Removed
+
+### Fixed
 
 ### Changed
 - Move `toLowerCasePropertyNames` inside `Utils.ts` and add test coverage.
+- Modified browser demo to make video preferences easier to test by adding explicit configuration replacing non-functional pin button
```

Comment:
- The browser demo now offers a configuration menu on each video tile. This menu replaces the 'Pin' button, which previously set the priority to X. The new configuration menu allows the user to specify the desired video quality and priority, which will be respected by simulcast and priority downlink policies. This is useful for testing or to demonstrate the behavior of those policies.